<launch>

    <arg name="rviz" default="true"/>

    <node
            pkg="btc_loop"
            type="load_cloud_show"
            name="load_cloud_show"
            output="screen"
    />
    <arg name="arg1" default="0.55"/>
    <param name="icp_threshold" value="$(arg arg1)"/>
    <param name="is_rgb" type="bool" value="false"/>
    <param name="is_body" type="bool" value="false"/>
    <param name="data_num" type="int" value="3"/>
    <param name="calc_gt_enable" type="bool" value="false"/>
    <param name="data_name" type="string" value="1"/>
    <param name="setting_path" type="string" value="$(find btc_loop)/config/config_multi_mapping.yaml"/>
    <param name="root_dir" type="string" value="/home/Downloads/test_data/"/>
    <param name="load_dir" type="string" value="/home/Downloads/test_data/"/>
    <param name="loop_gt_file" type="string" value="/home/weihairuo/bag/pose_correct/1.txt"/>

    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz"
              args="-d $(find btc_loop)/rviz_cfg/multi_section.rviz"/>
    </group>

</launch>